dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div class="form-field">
+    <div data-test="dataPointToggle" class="form-field border-none vertical-middle">
+      <InputSwitch
+        data-test="dataPointToggleButton"
+        inputId="dataPointIsAvailableSwitch"
+        @click="this.isItActive = false"
+        v-model="isItActive"
+      />
+      <h5 data-test="dataPointToggleTitle" class="m-2">
+        {{ label }}
+      </h5>
+    </div>
+
+    <FormKit type="group" :name="name" v-if="isItActive">
+      <div>
+        <div class="form-field border-none">
+          <NaceCodeFormFieldBindData
+            label="Definition Product Type/Service"
+            description="Define the procured product types/services per category (own operations)"
+            name="procuredProductTypesAndServicesNaceCodes"
+            v-model:selectedNaceCodesBind="procuredProductTypesAndServicesNaceCodesValue"
+          ></NaceCodeFormFieldBindData>
```

Comment:
We could close the tag right away: `<NaceCodeFormFieldBindData ... />

Reply:
Done.

Reply:
👍